src/nft.json

```diff
@@ -1903,10 +1903,6 @@
           "match": "0[xX][A-Fa-f0-9]+\\b",
           "name": "constant.numeric.hexadecimal.nft"
         },
-        {
-          "match": "\\b\"[^\"]*\"\\b",
```

Comment:
I think it would work to just change it to `\\b\"*\"\\b`

![image](https://user-images.githubusercontent.com/1681525/151211227-67f4a06c-493c-472d-93ed-aa7043d4c2a4.png)

Could you check on your end, and it it works fine, re-introduce the line, but with `\\b\"*\"\\b` as the match pattern?

Reply:
Hi. The change you suggest has the same effect for syntax highlighting in the editor. But I think this because the new regex only matches one or more consecutive quotation marks. So it basically matches nothing and therfore does not interfere with other rules. I've updated my branch accordingly.